SpatialGDK/Source/SpatialGDKEditorToolbar/Private/SpatialGDKEditorToolbar.cpp

```diff
@@ -754,14 +754,31 @@ bool FSpatialGDKEditorToolbarModule::FetchInspectorBinaryWrapper(FString Inspect
 {
 	bFetchingInspectorBinary = true;
 
-	const bool bSuccess =
-		SpatialCommandUtils::FetchInspectorBinary(InspectorVersion, GetDefault<USpatialGDKSettings>()->IsRunningInChina());
+	bool bSuccess = SpatialCommandUtils::FetchInspectorBinary(InspectorVersion, GetDefault<USpatialGDKSettings>()->IsRunningInChina());
 
 	if (!bSuccess)
 	{
 		UE_LOG(LogSpatialGDKEditorToolbar, Error, TEXT("Could not fetch the Inspector for version %s"), *InspectorVersion);
 		OnShowFailedNotification(TEXT("Failed to fetch local inspector!"));
+
```

Comment:
This newline appears to be different from the other PR, may want to make them the same to avoid merge conflicts.